packages/react/runtime/lazy/import.js

```diff
@@ -19,19 +19,102 @@ import {
 } from './target';
 
 Object.defineProperty(target, sExportsReact, {
-  value: ReactAPIs,
+  value: {
+    Children: ReactAPIs.Children,
```

Comment:
Would this create extra member expression and slow down the execution of `main-thread.js`?